server/closure.js

```diff
@@ -0,0 +1,22 @@
+function test() {
```

Comment:
I think it's great to save this file, but perhaps they should be collected in a separate folder